### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by tylerpendleton**
This has updates to all of the fonts.  Also updates the css of the topic cards and forms.
screen shot 2015-09-11 at 2 53 07 am

### CODE REVIEW

File: app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -28,22 +29,70 @@ body {
...
+  padding-top: -10px;
+}
+
 .ice {
   height: 350px;
   margin-top: 5px;
-  padding: 10px;
-  box-shadow: 2px 1px 5px #ccc;
+  padding: 5px;
+  border-style: solid;
```
Comment (colinrubbert) [inline app/assets/stylesheets/main.scss:63]: Refactor `border-style`, `border-width`, and `border-color` to
 `border: 1px solid rgba(0,0,0,0.1);`

File: app/controllers/talks_controller.rb [hunk 2]
```diff
@@ -38,7 +38,18 @@ def destroy
 
   def upvote
```
Comment (colinrubbert) [inline app/controllers/talks_controller.rb:39]: Anyway you can figure out how to put the `vote` method in the model? That would be the best, the controller is getting a little bloated. As a best practice we should figure out how to keep our controller skinny.

File: app/controllers/talks_controller.rb [hunk 3]
```diff
@@ -38,7 +38,18 @@ def destroy
...
+
+    # Determines whether or not a user has upvoted the current topic
+    @talk.votes.each do |x|
+      if x.user_id == current_user.id
+        @newvote = false
+      end
+    end
+
+    if @newvote
+      @talk.votes.create(:user_id => current_user.id)
```
Comment (colinrubbert) [inline app/controllers/talks_controller.rb:51]: Please use Ruby 2.0+ style guides (hash rockets have been deprecated). Let's keep the code uniformed so please change to the current style.
`@talk.votes.create(user_id: current_user.id)`

File: test/controllers/talks_controller_test.rb [hunk 4]
```diff
@@ -1,7 +1,41 @@
...
-  #   assert true
-  # end
+
+  test "no user upvote test" do
+    # create talk to collect votes
+    talk = FactoryGirl.create(:topic)
+
+    # ensure no votes pass into talk when no user is signed in
+    assert_no_difference "Talk.find(talk.id).votes.count" do
+      post :upvote, :id => talk.id
```
Comment (colinrubbert) [inline test/controllers/talks_controller_test.rb:11]: Same thing here with `=>`

File: test/controllers/talks_controller_test.rb [hunk 5]
```diff
@@ -1,7 +1,41 @@
...
+    # create talk to collect votes
+    talk = FactoryGirl.create(:topic)
+
+    # create user and sign in
+    user = FactoryGirl.create(:user)
+    sign_in user
+
+    # first vote should go through and redirect to home page
+    assert_difference "Talk.find(talk.id).votes.count" do
+      post :upvote, :id => talk.id
```
Comment (colinrubbert) [inline test/controllers/talks_controller_test.rb:28]: and here `=>`

File: test/controllers/talks_controller_test.rb [hunk 6]
```diff
@@ -1,7 +1,41 @@
...
+    end
+
+    assert_redirected_to talks_path
+
+    # second vote should not go through and should still redirect to home page
+    assert_no_difference "Talk.find(talk.id).votes.count" do
+      post :upvote, :id => talk.id
```
Comment (colinrubbert) [inline test/controllers/talks_controller_test.rb:35]: and here `=>`

**Comment by colinrubbert** [comment]
Still have some styling to work out for mobile. Might have to write custom media queries. 
firehose_lightning_screenshot

**Comment by colinrubbert** [comment]
@tpendleton1693 most of the things I made comments on are pretty much nitpicking but when you get into a real production application it is important to have the same styles and flow of the application. I would really love to see the `Vote` method in the model so we should figure that out. 

A note for future contributions be careful of creating branches off of branches that haven't been merged yet. Now we have two separate PRs with the same code and then one of those has additional styling changes. The result of which means that I will have accept it all or not accept any so just be conscious of that moving forward. It really isn't a huge deal in this situation but something to keep in mind moving forward. Other than the few nitpicking things I'd like you to take care of and resubmit, it looks pretty good. Keep it up!

**Comment by tylerpendleton** [comment]
I'm going to start over with the style redesign and submit a pull request when it's ready.  I'll make sure to test in mobile and take that into account this time.  Totally spaced last time.

**Comment by colinrubbert** [comment]
@tpendleton1693 I'm glad to hear you're going to move forward with them, they look great. I have a suggestion for mobile. The cards right now have a height hardcoded for the simple fact of making everything uniform in a tiled manner and I think that looks really good so what you could do is make a mediaquery for cell phones and remove that height restraint and that should actually solve the problem all together or at least it's something to think about. I was going to toy around with that idea before but just ran out of time. 

Anyways looking forward to what you come up with.

**Pull request closed without merging**